src/pspm_sf_dcm.m

```diff
@@ -143,7 +153,7 @@
 end
 miss_epoch = [ymissing_start(:),ymissing_end(:)];
 flag_missing_too_long = 0;
-if any(diff(miss_epoch, 1, 2)/sr > options.missingthresh)
+if any(diff(miss_epoch, 1, 2)/model.sr > options.missingthresh)
```

Comment:
if there are nan in the data, they need to be interpolated now, to avoid errors in VBA?

Reply:
Yes that is right. The idea was not implemented because we previously discussed to use `options.isYout` of `VBA_NLStateSpaceModel` to indicate missing epochs. However one of our recent issues said they were not always considered.. Therefore it is sensible to perform interpolation before using VBA functions.